commit 2ac7069ad7647cd1d9ca5b08765a1e116e13cdc4
Author: Alexandre Ghiti <alex@ghiti.fr>
Date:   Thu Dec 16 10:44:23 2021 +0100

    Documentation, arch: Remove leftovers from CIFS_WEAK_PW_HASH
    
    This config was removed so remove all references to it.
    
    Fixes: 76a3c92ec9e0 ("cifs: remove support for NTLM and weaker authentication algorithms")
    Signed-off-by: Alexandre Ghiti <alexandre.ghiti@canonical.com>
    Reviewed-by: Steve French <smfrench@gmail.com>
    Acked-by: Arnd Bergmann <arnd@arndb.de> [arch/arm/configs]
    Acked-by: Thomas Bogendoerfer <tsbogend@alpha.franken.de>
    Signed-off-by: Arnd Bergmann <arnd@arndb.de>

diff --git a/arch/arm/configs/cm_x300_defconfig b/arch/arm/configs/cm_x300_defconfig
index 502a9d870ca4..45769d0ddd4e 100644
--- a/arch/arm/configs/cm_x300_defconfig
+++ b/arch/arm/configs/cm_x300_defconfig
@@ -146,7 +146,6 @@ CONFIG_NFS_V3_ACL=y
 CONFIG_NFS_V4=y
 CONFIG_ROOT_NFS=y
 CONFIG_CIFS=m
-CONFIG_CIFS_WEAK_PW_HASH=y
 CONFIG_PARTITION_ADVANCED=y
 CONFIG_NLS_CODEPAGE_437=m
 CONFIG_NLS_ISO8859_1=m